components/brave_wallet_ui/utils/buy-asset-url.ts

```diff
@@ -0,0 +1,17 @@
+import { NetworkOptionsType, AssetOptionType, UserAccountType } from '../constants/types'
+export function BuyAssetUrl (network: NetworkOptionsType, asset: AssetOptionType, account: UserAccountType, buyAmount: string) {
+  switch (network.id) {
+    case 0:
```

Comment:
We don't have an enum for networks?

Reply:
We do as of this PR that just landed https://github.com/brave/brave-core/pull/9564/files#diff-349a8ec8ee1cea7e375455b4466c142dd7862c553ad883a94f50fe950171663fR213
Will have this setup in my next PR for this issue here https://github.com/brave/brave-browser/issues/17243